docfx_project/ziti/clients/tunneler-overview.md

```diff
@@ -1,4 +1,4 @@
-A tunneler is purpose-built software designed to connect applications which are not Ziti-aware to the Ziti Network.
-Ziti provides tunnelers for each major operating system.
+
+An OpenZiti Tunneler is purpose-built software designed to connect applications which are not Ziti-aware to the OpenZiti Network. Ziti Tunnellers allow pre-existing, unmodified TCP/IP applications to access or provide services on a secure OpenZiti Network. An OpenZiti Tunneler cannot secure the communication between itself and the TCP/IP client or server application.

```

Comment:
I'm sorry but everything after the lead is redundant information to me. Not to mention that it doesn't mention UDP... it's not just TCP. A tunneler bridges/connects/enables non-ziti-embedded software to a ziti network. all the rest is just extra words. This section doesn't need these extra words

"Ziti Tunnellers allow pre-existing, unmodified TCP/IP applications to access or provide services on a secure OpenZiti Network. An OpenZiti Tunneler cannot secure the communication between itself and the TCP/IP client or server application"

Reply:
Will send commit in https://github.com/openziti/ziti-doc/pull/138